stable/kommander/charts/kommander-thanos/values.yaml

```diff
@@ -45,10 +47,18 @@ thanos:
     - "--grpc-client-tls-ca=/etc/certs/ca.crt"
     - "--grpc-client-server-name=server.thanos.localhost.localdomain"
     certSecretName: kommander-thanos-client-tls
+    labels:
+      app.kubernetes.io/parent-component: kommander-thanos
+    grpc:
+      service:
+        matchLabels:
+          app.kubernetes.io/parent-component: kommander-thanos
```

Comment:
I haven't seen the `app.kubernetes.io/parent-component` label before, and googling it doesn't turn up anything. Is this ours? It won't break anything but we probably shouldn't use the `app.kubernetes.io` namespace for labels we come up with.

Is there a reason the existing selectors aren't sufficient? Maybe I'm missing something. I'd expect selecting by chart release name would be enough to prevent collisions with other thanos deployments: https://github.com/banzaicloud/banzai-charts/blob/3a18d063d72fd21d7f190f2b1be1e462d31f513a/thanos/templates/query-service.yaml#L25

Reply:
It is one that I came up with, couldn't find a standard one to reflect what I needed, I can see see about changing it to a mesosphere one (wasn't sure what the convention was).

The existing selectors are sufficient, I guess this was me making it extra clear where these components were coming from since the `name` label atm is set to `kommander-kubeaddons` which is pretty general wrt the kommander chart vs kommander-thanos subchart. But yeah, the matchLabels aren't required in this case, this was just me being a nitpick. If it's excessive (and also currently not very consistent since we can't define the same matchLabels on deployment) I'm happy to remove them too, don't feel too strongly about it either way. I can just leave this as a general thanos upgrade in that case.

Reply:
If the new match labels don't actually help the service match with its intended pods them IMO we should drop them. That also saves us the need to come up with label name.

Reply:
sgtm I shall remove! simplifies this pr a bunch :)